featuretools/tests/computational_backend/test_calculate_feature_matrix.py

```diff
@@ -917,13 +918,12 @@ def test_dask_persisted_es(es, capsys):
 
 class TestCreateClientAndCluster(object):
     def test_user_cluster_as_string(self, monkeypatch):
-        monkeypatch.setitem(create_client_and_cluster.__globals__,
-                            'LocalCluster',
-                            mock_cluster)
-        monkeypatch.setitem(create_client_and_cluster.__globals__,
-                            'Client',
-                            MockClient)
+        from featuretools.computational_backends import utils
 
+        def mock_get_client_cluster():
```

Comment:
MockClient and mock_cluster are only used for these tets so we can move this function to that utils file and import it instead.

Reply:
Fixed.